Applications/spire/source/time_and_sales/time_and_sales_properties_dialog.cpp

```diff
@@ -397,9 +382,9 @@ void time_and_sales_properties_dialog::update_colors(int band_index) {
   m_band_list->setStyleSheet(m_band_list_stylesheet + selected_stylesheet);
 }
 
-void time_and_sales_properties_dialog::update_font_preview_stylesheet() {
-  m_font_preview_label->setStyleSheet(QString(R"(
-    background-color: white;
-    border: %1px solid #C8C8C8 %2px solid #C8C8C8;
-    )").arg(scale_height(1)).arg(scale_width(1)));
+void time_and_sales_properties_dialog::update_band_list_font(
+    const QFont& font) {
+  for(auto i = 0; i < m_band_list->count(); ++i) {
```

Comment:
Can you look into whether this can be set on the widget itself rather than each individual item? The item's should not each be setting their own font but rather be using the parent window's default font.

Reply:
The only way to set it on the list widget itself is to use a stylesheet.

Reply:
Actually, might be wrong on that.